packages/features/schedules/lib/use-schedule/useSlotsForDate.ts

```diff
@@ -29,3 +33,15 @@ export const useSlotsForMultipleDates = (dates: (string | null)[], slots?: Slots
 
   return slotsForDates;
 };
+
+export const useSlotsForAvailableDates = (slots: slot[]) => {
+  const slotsForDates = useMemo(() => {
+    if (typeof slots === "undefined") return [];
+    return slots.map((slot) => ({
+      date: dayjs(slot[0].time).format("YYYY-MM-DD"),
+      slots: slot
+    }));
+  }, [slots]);
```

Comment:
getting this warning 
<img width="349" alt="Screenshot 2023-06-29 at 12 39 23 AM" src="https://github.com/calcom/cal.com/assets/53316345/5521ce73-57b6-449d-8f1a-b1ead3ca417b">

Reply:
it's bcz of frequent change in extraDays, it seems.

Reply:
https://github.com/calcom/cal.com/assets/123581387/0469c292-0146-40cd-9742-bd6c49d11ea6

@Udit-takkar FYI, Analysed and fixed that issue